### PR TITLE
Added rotational movement

### PR DESCRIPTION
**Pull request by samfolio**
I changed the movement to involve rotating left and right so it feels more shiplike. I also changed the temp sprite.

### CODE REVIEW

File: .godot/editor/editor_layout.cfg [hunk 1]
```diff
@@ -32,13 +32,13 @@ current_scene="res://scenes/player.tscn"
...
 [ScriptEditor]
 
-open_scripts=["res://scripts/player.gd", "res://README.md"]
-selected_script=""
-open_help=["@GlobalScope"]
+open_scripts=["res://scripts/player.gd", "res://scenes/player.gd", "res://README.md"]
+selected_script="res://scripts/player.gd"
+open_help=["@GlobalScope", "CanvasLayer", "float", "Node2D"]
 script_split_offset=70
 list_split_offset=0
```
Comment (vladngogo) [inline .godot/editor/editor_layout.cfg]: should probably add this to .gitignore this seems like noise

File: .godot/editor/filesystem_cache8 [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 ShipTemp.png::CompressedTexture2D::2383864839893551177::1715025669::1715025669::1::::<><>::
-::res://scenes/::1715025595
-game.tscn::PackedScene::2559903341372124374::1715025595::0::1::::<><>::
-::res://scripts/::1715025177
+::res://scenes/::1715314260
+game.tscn::PackedScene::2559903341372124374::1715314055::0::1::::<><>::uid://dklbkbnq1053w::::res://scenes/player.tscn
+player.gd::GDScript::-1::1715313656::0::1::::<>RefCounted<>::
+player.tscn::PackedScene::7883937291054637294::1715314056::0::1::::<><>::uid://bdiwt4xjspmx0::::res://assets/sprites/BoatSpriteSheet.png<>res://scripts/player.gd
+::res://scripts/::1715313269
+player.gd::GDScript::-1::1715313269::0::1::::<>CharacterBody2D<>::
```
Comment (vladngogo) [inline .godot/editor/filesystem_cache8:70]: .gitignore? are these changes material?

File: scripts/player.gd [hunk 3]
```diff
@@ -1,33 +1,53 @@
...
-	# As good practice, you should replace UI actions with custom gameplay actions.
-	var direction = Input.get_axis("ui_left", "ui_right")
-	if direction:
-		velocity.x = direction * SPEED
-		print("Direction is: " + direction);
-		print($"Speed is: {SPEED}");
+	player_movement(delta)
+
+func get_input():
+	rotation_direction = Input.get_axis("ui_left", "ui_right")
```
Comment (vladngogo) [inline scripts/player.gd:26]: where are ui_left and ui_right defined?

**Comment by vladngogo** [comment]
this looks good, just wondering if we can suppress some of these noisy changes, could we investigate whether these changes are material or if we're just pulling in cached data from individual settings on our machines. Ideally a change should be pretty limited, seems like a lot of noise right now

File: scripts/player.gd (hunk 3 above)
Comment (samfolio) [reply scripts/player.gd:26]: They don't need to be defined until we need to change the movement keys. The engine knows that ui_direction means to use that arrow key.

**Review by vladngogo — APPROVED** [review]

**Pull request merged**